### DISCUSSION TITLE
Outdated responses from /sync/playback endpoint

### DISCUSSION THREAD

**Discussion by sharkykh** [trakt/trakt-api#341] [Q&A] [answered]
I make calls to [`GET /sync/playback`](https://trakt.docs.apiary.io/#reference/sync/playback/get-playback-progress) and the results I get are progress items that are **no longer valid**.
Meaning, I already called [`DELETE /sync/playback/:id`](https://trakt.docs.apiary.io/#reference/sync/remove-playback/remove-a-playback-item) to remove them, but they still come up in the results.

Example requests - this contains calls to get playback and then delete one of the results (you can ignore the `/users/me/watching` call)
Only redacted access and API keys from the requests.
[trakt.har.zip](https://github.com/trakt/api-help/files/8546723/trakt.har.zip)
(Can be viewed using DevTools or [HAR Analyzer](https://toolbox.googleapps.com/apps/har_analyzer/))

I had received feedback from another user that they are getting "ghost" playback progress items, too, so I figured this is not just my own account, and maybe a bug.

If you could look into that please, and thank you 🙂

**2022-04-25:** I noticed that after some time the results are updated. Is it some kind of cache that should not be there? `/users/me/watching` still seems to be okay in that regard...

**Comment by jonathanantoine** [comment]
I have the same report from my users

**Comment by rectifyer** [comment, marked as answer]
I'll have a fix out for this tomorrow. The cache key needed an update.

**Reply by sharkykh** [reply]
Seems to be working correctly now. Thank you very much!
